src/components/views/game/room/RoomInfoContainer.tsx

```diff
@@ -4,32 +4,50 @@ import { useCurrentGame } from "../../../../hooks/api/useCurrentGame"
 import Button from "@mui/material/Button"
```

Comment:
With the sopra resolution (1366x768), this does not look good:
![grafik](https://user-images.githubusercontent.com/1506818/169690428-0bc6789f-b863-4eab-9213-a6739c12da3f.png)

Reply:
Fixed this! It looks a bit smaller on the "normal" resolution now, but at least the SoPra resolution looks presentable now 😂